README.md

```diff
@@ -92,7 +92,21 @@ A wildcard stands for all the non-specified HTTP methods.
 
 ##### Key: `<cache-header-name>` 
 
-A http cache header name (`Cache-control`, `Expires`, etc.).
+A http cache header name (`cache-control`, `expires`, `etag` etc.).
```

Comment:
- `s/A http/An HTTP/`
- use TitleCase for the header names. Lookups are case insensitive, but users are more accustomed to standard casing.